### PR TITLE
[Synapse]add new apis of Pause/Resume for Synapse Link

### LINKED ISSUE #21421: [Azure Synapse Platform Service - Synapse Link] API Review
New API Review meeting has been requested.

**Service Name**: Azure Synapse Platform Service - Synapse Link
**Review Created By**: Zhicheng Yang
**Review Date**: 11/17/2022 4:00 PM PT
**PR**: https://github.com/Azure/azure-rest-api-specs/pull/21240
**Hero Scenarios Link**: Not Provided
**Core Concepts Doc Link**: Not Provided

**Description**: 

Detailed meeting information and documents provided can be accessed [here](https://web.powerapps.com/apps/e41c69de-ef96-44ba-b287-0e012e7c13ae?view=edit&event=a06c9c38-e55b-ed11-9562-000d3a341bd5)
For more information that will help prepare you for this review, the requirements, and office hours, visit the documentation [here](https://aka.ms/azsdk/onboarding/restapischedule)

### PR DESCRIPTION
**Pull request by ZhichengYang-MS**
# Data Plane API - Pull Request
Add new apis of Pause and Resume for Synapse Link

## API Info: The Basics
Most of the information about your service should be captured in the issue that serves as your [*engagement record*](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/271/Azure-REST-API-Stewardship?anchor=rest-api-stewardship-process).

* Link to engagement record issue:

Is this review for (select one):

- [ ] a private preview
- [x] a public preview
- [ ] GA release

### Change Scope
<sup>This section will help us focus on the specific parts of your API that are new or have been modified. <br/>Please share a link to the design document for the new APIs, a link to the previous Open API document (swagger) if applicable, and the root paths that have been updated. </sup>
* Design Document: [Design of Synapse Link Pause and Resume](https://microsoftapc-my.sharepoint.com/:w:/g/personal/zhichengyang_microsoft_com/Eb7quG0SrShPgRRbR04Ki5IBDSVj2RqaAHY2g1qf-DqoVg?e=ucBUeq)
* Previous Open API Doc:
* Updated paths: 'specification\synapse\data-plane\Microsoft.Synapse\preview\2022-11-01-preview' and 'specification\synapse\data-plane\readme.md'

## ❔Got questions? Need additional info?? We are here to help!

<details>
  <summary> Contact us!</summary>

The [Azure API Review Board](https://aka.ms/azapi) is dedicated to helping you create amazing APIs. You can read about our mission and learn more about our process on our [wiki](https://aka.ms/azapi).
* 💬 [Teams Channel](https://teams.microsoft.com/l/channel/19%3a3ebb18fded0e47938f998e196a52952f%40thread.tacv2/General?groupId=1a10b50c-e870-4fe0-8483-bf5542a8d2d8&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47)
* 💌 [email](mailto://azureapirbcore@microsoft.com)

</details>

<details>
  <summary>Click here for links to tools, specs, guidelines & other good stuff</summary>

### Tooling
 * [Open API validation tools](https://aka.ms/swaggertools) were run on this PR. Go here to see [how to fix errors](https://aka.ms/ci-fix)
 * [Spectral Linting](https://aka.ms/style)
 * [Open API Hub](https://aka.ms/openapiportal)

### Guidelines & Specifications
 * [Azure REST API Guidelines](https://aka.ms/guidelines)
 * [OpenAPI Style Guidelines](https://aka.ms/style)
 * [Azure Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy)

### Helpful Links
 * [Azure DevTools Wiki](https://aka.ms/azapi)

</details>

fix https://github.com/Azure/azure-rest-api-specs/issues/21421

### CODE REVIEW

File: specification/synapse/data-plane/Microsoft.Synapse/preview/2022-11-01-preview/linkConnections.json [hunk 1]
```diff
@@ -0,0 +1,1043 @@
...
+        "parameters": [
+          {
+            "$ref": "#/parameters/LinkConnectionName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
```
Comment (weidongxu-microsoft) [inline specification/synapse/data-plane/Microsoft.Synapse/preview/2022-11-01-preview/linkConnections.json:461]: why it is not 204?

File: specification/synapse/data-plane/Microsoft.Synapse/preview/2022-11-01-preview/linkConnections.json [hunk 2]
```diff
@@ -0,0 +1,1043 @@
...
+          },
+          "default": {
+            "description": "An error response received from the Azure Synapse service.",
+            "schema": {
+              "$ref": "../../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "LinkConnection_Start": {
```
Comment (weidongxu-microsoft) [inline specification/synapse/data-plane/Microsoft.Synapse/preview/2022-11-01-preview/linkConnections.json:472]: LinkConnection_Start ? Pasue?

File: specification/synapse/data-plane/Microsoft.Synapse/preview/2022-11-01-preview/linkConnections.json (hunk 1 above)
Comment (ZhichengYang-MS) [reply specification/synapse/data-plane/Microsoft.Synapse/preview/2022-11-01-preview/linkConnections.json:461]: We want to keep consistent to existing APIs to not bring breaking changes

File: specification/synapse/data-plane/Microsoft.Synapse/preview/2022-11-01-preview/linkConnections.json (hunk 2 above)
Comment (ZhichengYang-MS) [reply specification/synapse/data-plane/Microsoft.Synapse/preview/2022-11-01-preview/linkConnections.json]: changed to _pause and _resume

File: specification/synapse/data-plane/Microsoft.Synapse/preview/2022-12-01-preview/linkConnections.json [hunk 3]
```diff
@@ -0,0 +1,1043 @@
...
+    "LinkTableStatus": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "ID provided by the client"
+        },
+        "status": {
+          "type": "string",
+          "description": "Link table status, please refer to this [articles](https://learn.microsoft.com/en-us/azure/synapse-analytics/synapse-link/sql-database-synapse-link#monitoring) for details."
```
Comment (weidongxu-microsoft) [inline specification/synapse/data-plane/Microsoft.Synapse/preview/2022-12-01-preview/linkConnections.json:872]: link should not contain "en-us"

File: specification/synapse/data-plane/Microsoft.Synapse/preview/2022-12-01-preview/linkConnections.json [hunk 4]
```diff
@@ -0,0 +1,1043 @@
...
+        "value": {
+          "type": "array",
+          "description": "List link table value",
+          "items": {
+            "$ref": "#/definitions/LinkTableResource"
+          }
+        }
+      }
+    },
+    "LinkConnectionResource": {
```
Comment (weidongxu-microsoft) [inline specification/synapse/data-plane/Microsoft.Synapse/preview/2022-12-01-preview/linkConnections.json:544]: most of the models do not have "description"

they will be required in GA.

File: specification/synapse/data-plane/Microsoft.Synapse/preview/2022-12-01-preview/linkConnections.json (hunk 3 above)
Comment (ZhichengYang-MS) [reply specification/synapse/data-plane/Microsoft.Synapse/preview/2022-12-01-preview/linkConnections.json]: removed

File: specification/synapse/data-plane/Microsoft.Synapse/preview/2022-12-01-preview/linkConnections.json (hunk 4 above)
Comment (ZhichengYang-MS) [reply specification/synapse/data-plane/Microsoft.Synapse/preview/2022-12-01-preview/linkConnections.json:544]: will review and add missing description in GA PR

**Review by weidongxu-microsoft — APPROVED** [review]

**Pull request merged**